net/ftpproxy/Makefile

```diff
@@ -0,0 +1,6 @@
+PLUGIN_NAME=       ftpproxy
+PLUGIN_VERSION=    1.0
+PLUGIN_COMMENT=    Control ftp-proxy processes
+PLUGIN_MAINTAINER= frank.brendel@eurolog.com
+
```

Comment:
Also set the following to name the final package "ospriv-.." instead of "os-.." in the meantime:

PLUGIN_PRIVATE= yes